20240917-BR-resiliency-error-code-retries.md

```diff
@@ -0,0 +1,145 @@
+# Resiliency Policy Error Code Retries
+
+* Author(s): Anton Troshin (@antontroshin), Taction (@taction)
+* Updated: 2024-09-18
+
+## Overview
+
+This is a design proposal to provide additional functionality for Dapr Resiliency Policy Retries to be able to enforce policy only on specific response error codes.
+It only focuses on the `retries` (https://docs.dapr.io/operations/resiliency/policies/#retries) part of the policy.
+
+## Background
+
+In some applications, some status codes may be used to indicate the business error, and retrying the operation might not be necessary or otherwise desirable.
+Customizing retry behavior will allow a more granular way to handle error codes that suit each use case.
+Currently, all errors are retried when the policy is applied.
+Some status codes are not retryable, and subsequent calls will result in the same error. Avoiding these retry calls will reduce the overall number of requests, traffic, and errors.
+
+## Related Items
+
+https://github.com/dapr/dapr/issues/6683
+https://github.com/dapr/dapr/issues/6428
+https://github.com/dapr/dapr/issues/7697
+
+PR:
+https://github.com/dapr/dapr/pull/7132
+
+Docs:
+https://github.com/dapr/docs/issues/4254
+https://github.com/dapr/docs/issues/3859
+
+## Expectations and alternatives
+
+* What is in scope for this proposal?
+  - HTTP and gRPC Service Invocation, direct and proxied
+  - Bindings
+  - Pub/Sub
```

Comment:
How is pubsub included here- I thought this was for bare HTTP and gRPC. Pubsub clients use different clients AFAIK